pages/api/auth/[...nextauth].ts

```diff
@@ -47,8 +63,10 @@ export const authOptions: AuthOptions = {
     maxAge: 30 * 24 * 60 * 60, // 30 days
   },
   jwt: {
-    maxAge: 30 * 24 * 60 * 60, // 30 days
+    secret: process.env.NEXTAUTH_SECRET, 
```

Comment:
interesting,maybe we were missing the secret before. where did you see that this might be required? somewhere in the next auth docs?